### ISSUE TITLE
cobbler buildiso fails on Ubuntu 20.04.2 LTS

### ISSUE THREAD

**Issue by mohantnr** [cobbler/cobbler#2576]
### Describe the bug

buildiso is failing with below error message 
```
cobbler buildiso
task started: 2021-03-07_193139_buildiso
task started (id=Build Iso, time=Sun Mar  7 19:31:39 2021)
running python triggers from /var/lib/cobbler/triggers/task/buildiso/pre/*
running shell triggers from /var/lib/cobbler/triggers/task/buildiso/pre/*
shell triggers finished successfully
using/creating buildisodir: /var/cache/cobbler/buildiso
building tree for isolinux
copying miscellaneous files
Exception occurred: <class 'ValueError'>
Exception value: too many values to unpack (expected 2)
Exception Info:
  File "/usr/local/lib/python3.8/dist-packages/cobbler/remote.py", line 95, in run
    rc = self._run(self)

  File "/usr/local/lib/python3.8/dist-packages/cobbler/remote.py", line 162, in runner
    self.remote.api.build_iso(

  File "/usr/local/lib/python3.8/dist-packages/cobbler/api.py", line 1627, in build_iso
    builder.run(

  File "/usr/local/lib/python3.8/dist-packages/cobbler/actions/buildiso.py", line 723, in run
    for k, v in file_copy_success:

!!! TASK FAILED !!!
```

### Version

````
Cobbler 3.2.1
````

### Additional context

cobbler build iso is failing with below message

```
# cobbler buildiso
task started: 2021-03-07_192854_buildiso
task started (id=Build Iso, time=Sun Mar  7 19:28:54 2021)
running python triggers from /var/lib/cobbler/triggers/task/buildiso/pre/*
running shell triggers from /var/lib/cobbler/triggers/task/buildiso/pre/*
shell triggers finished successfully
using/creating buildisodir: /var/cache/cobbler/buildiso
building tree for isolinux
copying miscellaneous files
Exception occurred: <class 'ValueError'>
Exception value: too many values to unpack (expected 2)
Exception Info:
  File "/usr/local/lib/python3.8/dist-packages/cobbler/remote.py", line 95, in run
    rc = self._run(self)

  File "/usr/local/lib/python3.8/dist-packages/cobbler/remote.py", line 162, in runner
    self.remote.api.build_iso(

  File "/usr/local/lib/python3.8/dist-packages/cobbler/api.py", line 1627, in build_iso
    builder.run(

  File "/usr/local/lib/python3.8/dist-packages/cobbler/actions/buildiso.py", line 723, in run
    for k, v in file_copy_success:

!!! TASK FAILED !!!

cobbler sync is completed 

root@hdrkairosvm203:/var/lib/cobbler/loaders# cobbler sync
task started: 2021-03-07_193027_sync
task started (id=Sync, time=Sun Mar  7 19:30:27 2021)
running python triggers from /var/lib/cobbler/triggers/task/sync/pre/*
running shell triggers from /var/lib/cobbler/triggers/task/sync/pre/*
shell triggers finished successfully
running pre-sync triggers
cleaning trees
removing: /var/www/cobbler/images/Ubuntu-20.04.2-LTS-casper-x86_64
removing: /srv/tftpboot/pxelinux.cfg/0A044ACC
removing: /srv/tftpboot/pxelinux.cfg/default
removing: /srv/tftpboot/grub/images
removing: /srv/tftpboot/grub/system
removing: /srv/tftpboot/grub/system_link
removing: /srv/tftpboot/grub/grub.cfg
removing: /srv/tftpboot/grub/local_efi.cfg
removing: /srv/tftpboot/grub/local_legacy.cfg
removing: /srv/tftpboot/grub/local_powerpc-ieee1275.cfg
removing: /srv/tftpboot/grub/x86_64_menu_items.cfg
removing: /srv/tftpboot/images/Ubuntu-20.04.2-LTS-casper-x86_64
copying bootloaders
running: ['rsync', '-rpt', '--copy-links', '--exclude=.cobbler_postun_cleanup', '/var/lib/cobbler/loaders/', '/srv/tftpboot']
received on stdout:
received on stderr:
running: ['rsync', '-rpt', '--copy-links', '--exclude=README.grubconfig', '/var/lib/cobbler/grub_config/', '/srv/tftpboot']
received on stdout:
received on stderr:
copying distros to tftpboot
copying files for distro: Ubuntu-20.04.2-LTS-casper-x86_64
running: /usr/bin/sha1sum /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/vmlinuz
received on stdout: 0a8cf1d80f04ecf2d76918dd772d742da686f18a  /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/vmlinuz

received on stderr:
trying cachelink /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/vmlinuz -> /srv/tftpboot/images/.link_cache/0a8cf1d80f04ecf2d76918dd772d742da686f18a -> /srv/tftpboot/images/Ubuntu-20.04.2-LTS-casper-x86_64/vmlinuz
running: /usr/bin/sha1sum /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/initrd
received on stdout: f23fc0c181223bc29ccadc6bf41e66ea005ed1bd  /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/initrd

received on stderr:
trying cachelink /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/initrd -> /srv/tftpboot/images/.link_cache/f23fc0c181223bc29ccadc6bf41e66ea005ed1bd -> /srv/tftpboot/images/Ubuntu-20.04.2-LTS-casper-x86_64/initrd
copying images
generating PXE configuration files
generating: /srv/tftpboot/pxelinux.cfg/0A044ACC
generating: /srv/tftpboot/grub/system/0A044ACC
generating PXE menu structure
copying files for distro: Ubuntu-20.04.2-LTS-casper-x86_64
trying hardlink /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/vmlinuz -> /var/www/cobbler/images/Ubuntu-20.04.2-LTS-casper-x86_64/vmlinuz
trying hardlink /var/www/cobbler/distro_mirror/Ubuntu-20.04.2-LTS/casper/initrd -> /var/www/cobbler/images/Ubuntu-20.04.2-LTS-casper-x86_64/initrd
Writing template files for Ubuntu-20.04.2-LTS-casper-x86_64
processing boot_files for distro: Ubuntu-20.04.2-LTS-casper-x86_64
cleaning link caches
running: find /srv/tftpboot/images/.link_cache -maxdepth 1 -type f -links 1 -exec rm -f '{}' ';'
received on stdout:
received on stderr:
running post-sync triggers
running python triggers from /var/lib/cobbler/triggers/sync/post/*
running python trigger cobbler.modules.sync_post_restart_services
running shell triggers from /var/lib/cobbler/triggers/sync/post/*
shell triggers finished successfully
running python triggers from /var/lib/cobbler/triggers/change/*
running python trigger cobbler.modules.scm_track
running python trigger cobbler.modules.managers.genders
running shell triggers from /var/lib/cobbler/triggers/change/*
shell triggers finished successfully
*** TASK COMPLETE ***
```

**Comment by mohantnr** [comment]
any workaround for this problem?

**Comment by SchoolGuy** [comment]
@mohantnr So far no. We are trying to investigate.

**Comment by aguerson** [comment]
https://github.com/cobbler/cobbler/issues/2697

**Comment by SchoolGuy** [comment]
Since I am refactoring the code for `buildiso` right now this will be closed with #2416
